docs/user-guide/api-mediation/configuration-logging.md

```diff
@@ -0,0 +1,104 @@
+# Customizing Zowe API Mediation Layer logging
+
+Zowe API Mediation Layer uses logback for its logging. You can customize the logging of Zowe API Mediation Layer by specifying the customized `logback.xml` file in `zowe.yaml` for each service separately.
+
+To change the default logback configuration file, set `components.<component>.logging.config` with a path to your `logback.xml`.
+
+* `component`  
+  Specifies one of the following services:
+    - `gateway`
+    - `discovery`
+    - `api-catalog`
+    - `caching-service`
+    - `cloud-gateway`
+    - `metrics-service`
```

Comment:
Are we talking about v3? In the case we should replace metrics-service with zaas and remove cloud-gateway. Could we create both versions?

Reply:
It's for V2. And yes, for V3 the list should be updated, but it will be different PR (same as differnt PR in api-layer), I guess, not to `docs-staging`, which is for V2, right now.